website/content/docs/connect/ca/vault.mdx

```diff
@@ -210,48 +210,79 @@ The following Vault policy allows Consul to use pre-existing PKI paths in Vault.
 Consul is granted read-only access to the PKI mount points and the Root CA, but is
 granted full control of the Intermediate or Leaf CA for Connect clients.
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+This example uses placeholder values for `RootPKIPath` and `IntermediatePKIPath`.
+Please update those placeholders to match the path values in the CA provider configuration.
 
 <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
 ```hcl
-# Existing PKI Mounts
-path "/sys/mounts" {
-  capabilities = [ "read" ]
-}
+# -----------------------------------------------------------------------------
+# Vault Managed PKI Mounts
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# Needed for Consul to read the PKI mounts already created in Vault
 
-path "/sys/mounts/connect_root" {
+path "/sys/mounts/<insert-RootPKIPath>" {
   capabilities = [ "read" ]
 }
 
-path "/sys/mounts/connect_inter" {
+path "/sys/mounts/<insert-IntermediatePKIPath>" {
   capabilities = [ "read" ]
 }
 
-# Needed for Consul 1.11+
-path "/sys/mounts/connect_inter/tune" {
+# Needed in Consul 1.11+ to update intermediate mount configuration
+path "/sys/mounts/<insert-IntermediatePKIPath>/tune" {
   capabilities = [ "update" ]
 }
 
-path "/connect_root/" {
+# -----------------------------------------------------------------------------
+# Needed for Consul to use the externally-managed, mounted PKI secrets engines
+
+path "/<insert-RootPKIPath>/" {
   capabilities = [ "read" ]
 }
 
-path "/connect_root/root/sign-intermediate" {
+path "/<insert-RootPKIPath>/root/sign-intermediate" {
   capabilities = [ "update" ]
 }
 
-path "/connect_inter/*" {
+path "/<insert-RootPKIPath>/*" {
   capabilities = [ "create", "read", "update", "delete", "list" ]
 }
 
+# Needed only when changing the CA provider from Vault with this RootPKIPath
+# to either:
+# 1. A different CA provider (such as Consul's built-in provider)
+# 2. Changing the RootPKIPath used by the Vault CA provider
+#
+# SECURITY NOTE: Because this is an extremely privileged endpoint,
+# We recommend only temporarily granting this permission if there's a need
+# to modify Consul's CA provider configuration as in (1) or (2).
+# That's why this permission is currently commented out.
+#
+# path "<insert-RootPKIPath>/root/sign-self-issued" {
```

Comment:
@rboyer : This is my attempt to strike a balance between documenting this necessary permission while optimizing for security.

I also reference this from the "modify CA" CLI and HTTP API docs.